scripts/htop_suppressions.valgrind

```diff
@@ -0,0 +1,63 @@
+{
+   <insert_a_suppression_name_here>
```

Comment:
It would be nice to have good suppression_names for the ones that should stay,
e.g. explain that ncurses retains memory and we cannot change that.
Not everybody is a valgrind expert :-).